css/style.css

```diff
@@ -0,0 +1,76 @@
+body {
+  background-image: url(../assets/img/image_geometry_1.png);
+  background-position: top left;
+  background-repeat: no-repeat;
+}
+
+a {
+  color: #000;
+  text-decoration: none;
+  font-size: 24px;
+}
+
+.name {
+  padding-left: 5%;
+  font-family: Playfair Display, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 7vw;
```

Comment:
I think you should stick to the units given in the template as using others requires proper conversion.